lib/tasks/export_dags.rake

```diff
@@ -15,16 +21,20 @@ task :export_dags, [:sample_id] => :environment do |_, args|
     end
   end
 
-  if args.sample_id?
+  if args.sample_id
```

Comment:
this line just didn't work for me until I changed it... `sample_id?` was always `nil`